helm/alfresco-content-services/values.yaml

```diff
@@ -773,9 +735,14 @@ alfresco-control-center:
     limits:
       cpu: "1"
       memory: "1024Mi"
-# -- Enable/Disable Alfresco Collaboration Connector for Microsoft 365
-ooi:
+alfresco-connector-ms365:
+  # -- Enable/Disable Alfresco Collaboration Connector for Microsoft 365
   enabled: false
+  image:
+    tag: 2.0.0
+  repository:
+    nameOverride: acs-alfresco-cs-repository
```

Comment:
this is making things working because the release name we test against is `acs` but it should not be set here, the chart should use a named template which include the release name to generate the final name

Reply:
From what I can see in the ms365 chart there is no support for expanding the repo address from a `nameOverride`. The only option we have for now is to pass `repository.host` & `repository.port` as values.

Previous chart was also using that simplistic approach with a static value. As @gionn pointed out, the default value will only work for the default release name `acs`we use in different places. That is lame but was already the case.

Make sure to:
* revert using host & port values
* make it clear in the documentation that in order to use ms365 connector one needs to pass the static name of the service